lib/rebay_api/listings/item.ex

```diff
@@ -0,0 +1,22 @@
+defmodule RebayApi.Listings.Item do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "items" do
+    field :category, :string
+    field :description, :string
+    field :end_date, :utc_datetime
+    field :image, :string
+    field :price, :float
```

Comment:
https://husobee.github.io/money/float/2016/09/23/never-use-floats-for-currency.html

Reply:
Done! See latest commit